combiner.yaml

```diff
@@ -46,50 +43,5 @@ services:
       dockerfile: components/client/Dockerfile
     working_dir: /app/client
     command: fedn run combiner -d ${FEDN_REDUCER_HOST} -p ${FEDN_REDUCER_PORT} -t ${FEDN_ALLIANCE_ADMIN_AUTH_TOKEN} -h combiner2 -i 12081 -n combiner2
-    volumes:
```

Comment:
For now, should we add a volume mount on host for the TempFileStorage to mitigate Combiner failure risk?